tests/python/gaia-ui-tests/gaiatest/apps/ftu/app.py

```diff
@@ -334,9 +334,20 @@ def a11y_click_next_to_welcome_browser_section(self):
             Wait(self.marionette).until(expected.element_present(
                 *self._section_welcome_browser_locator))))
 
+    @property
+    def is_statistics_enabled(self):
+        return self.marionette.find_element(
+            *self._statistic_checkbox_locator).is_selected()
+
     def tap_statistics_checkbox(self):
```

Comment:
Posted in the bug, but I see a difference between a UI operation "tap" and a semantic operation "toggle the option however, with a check"

That said, my naming sucked in terms of making that differentation. I followed the lead in Settings, and shouldn't have.

I've changed the naming in the new PR to make the difference clear.

As for changing or removing the other function, unless you grepped every possible test base based on this one, including MTBF and so forth that's not in our repo, that's usually not a great idea. Once you publish an interface, it's generally better to keep it. But in this case I would have anyway.